manifest.yaml

```diff
@@ -224,6 +229,10 @@
   BuildScript: ./Android/GoogleBillingClient/build.cake
   TriggerPaths: [ Android/GoogleBillingClient ]
   MacBuildTargets: [ libs, nuget]
+- Name: Dagger
```

Comment:
I don't think we need this as well, just the GoogleDagger above should be good enough.